README.md

```diff
@@ -6,6 +6,10 @@ This plugin checks, whether the current buffer contains an `import` or a `requir
 
 Note however, that this plugin does not provide its own CSS rules. You should have an up to date vim runtime (for CSS2 rules) and - as long as CSS3 rules are not included - an additional plugin for CSS3 rules. (I recommend [hail2u/vim-css3-syntax](https://github.com/hail2u/vim-css3-syntax) but feel free to use whatever you like.)
 
+**Note:** This plugin won't change the syntax if you start with an empty buffer and then add styled-components code, you will need to close the buffer and re-open it so that the styles apply.
```

Comment:
Albeit valuable information I think there should be some changes done:
- Don't make this Note bold. This unevenly distributes the attention towards this note (there is another equally important note above).
- This plugin won't change the syntax at all, it only highlights the syntax. I'd like it to say this as well.
- Make this paragraph two sentences. The first until the first comma and the subsequent part as the second.